webapp/channels/src/components/settings_sidebar/settings_sidebar.tsx

```diff
@@ -110,32 +113,39 @@ export default class SettingsSidebar extends React.PureComponent<Props> {
             pluginTabList = (
                 <>
                     <hr/>
-                    <li
-                        key={'plugin preferences heading'}
-                        role='heading'
-                        className={'header'}
+                    <div
+                        role='group'
+                        aria-labelledby='userSettingsModal.pluginPreferences.header'
                     >
-                        <FormattedMessage
-                            id={'userSettingsModal.pluginPreferences.header'}
-                            defaultMessage={'PLUGIN PREFERENCES'}
-                        />
-                    </li>
-                    {this.props.pluginTabs.map((tab, index) => this.renderTab(tab, index))}
+                        <div
+                            key={'plugin preferences heading'}
+                            role='heading'
+                            className={'header'}
+                            aria-level={3}
+                            id='userSettingsModal.pluginPreferences.header'
```

Comment:
normally we don't use this words dot-splitted format for defining the ids, this because dots can be confused with object property notation. I think we should follow the one word, maybe camel-cased or low dash notation.